### ISSUE TITLE
`always-return`: enable `ignoreLastCallback` by default

### ISSUE THREAD

**Issue by gurgunday** [eslint-community/eslint-plugin-promise#452]
### Description

I think the following is a perfectly valid use case, and I understand the argument against it: possible race conditions, async side effects, etc. But this is the nature of these callbacks, and, the rule is not enforced for `.catch` anyway, so we could argue the same thing for it too

In reality, this pattern is useful, and having it banned in the recommended config feels a little too intrusive for me

I can provide a PR

### Steps to Reproduce

```js
  import("/assets/path/deps/utils.js")
    .then((module) => {
      window.movePicCarousel = module.movePicCarousel;
    })
    .catch(console.error);
```

**Expected behavior:** [What you expect to happen]
No errors (like when `ignoreLastCallback: true`)

**Actual behavior:** [What actually happens]
Error

**Comment by SchroederSteffen** [comment]
+1 for this, would be great to see this changed.
